ui/appui-react/src/appui-react/frontstage/FrontstageDef.tsx

```diff
@@ -272,8 +291,14 @@ export class FrontstageDef {
   public async onActivated() {
     this.updateWidgetDefs();
 
-    if (this._contentGroupProvider && this._initialProps) {
-      this._contentGroup = await this._contentGroupProvider.provideContentGroup(this._initialProps);
+    const provider = this._contentGroupProvider;
+    if (provider && this._initialProps) {
+      if (provider.contentGroup) {
+        const config = toFrontstageConfig(this._initialProps);
```

Comment:
Since the frontstage can be activated more than once, should we save the config, so we can reuse it on subsequent calls ?

Reply:
It costs nothing to convert between Props and Config, let's keep this branch free.